homework/03LessonThree.js

```diff
@@ -79,7 +80,7 @@ var problemSevenAnswer;
 // What is the ternary operator?  How does it differ from normal conditional statements?
 //
 // Your Answer Goes Here:
-//
+// a ternary operator allows you to toggle back and forth between two vaules. they are a cleaner way to write code.
```

Comment:
Good answer but what initializes the toggle? What condition?

What case use would ternary operators not make sense for?